src/client/index.js

```diff
@@ -388,6 +389,31 @@ function mkcircuit(data, opts) {
     show_scale();
     monitorview.on('change:start', show_range);
     monitorview.on('change:pixelsPerTick', show_scale);
+
+
+    $('button[name=zoom-in]').click(e => {
+        // Zoom only if we already have a circuit on the paper
+        if (typeof paper.scale === 'function') {
```

Comment:
Maybe it's better to hide or disable the buttons instead of displaying clickable, but unusable buttons?

Reply:
Yes. This is a good point. I'm going to change it.

Reply:
Fixed in [this commit](https://github.com/tilk/digitaljs_online/pull/34/commits/7c3ebca2de5160fffce756a398be8ccb3791aec4).